components/Makefile

```diff
@@ -161,7 +161,9 @@ $(COMPONENT_DIRS):	$(WS_LOGS) setup FORCE
 
 incorporation:
 	$(eval COMPONENT_REVISION := $(shell git rev-list HEAD --count))
-	$(WS_TOOLS)/userland-incorporator --repository $(PKG_REPO) \
+	$(eval INCORPORATION_REPO := $(or $(INCORPORATION_REPO),$(PKG_REPO)))
```

Comment:
Why not to do it this way (outside the `incorporation` target)?
```
INCORPORATION_REPO = $(PKG_REPO)
```

Reply:
That should be ?=

Reply:
We do have many cases that are only there for a single step (or target) only.  For example, `COMPONENTS_IGNORE` above in this file.